common/build/build-common/api-extractor-base.json

```diff
@@ -0,0 +1,218 @@
+{
+	"$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+
+	"compiler": {
+		"tsconfigFilePath": "<projectFolder>/tsconfig.json"
+	},
+
+	/**
+	 * Specifies the .d.ts file to be used as the starting point for analysis. API Extractor analyzes the symbols exported by this module.
+	 * Can be overridden as necessary for packages with different build structures.
+	 *
+	 * See: <https://api-extractor.com/pages/configs/api-extractor_json/#mainentrypointfilepath>
+	 */
+	"mainEntryPointFilePath": "<projectFolder>/dist/index.d.ts",
+
+	/**
+	 * Configures how the API report file (*.api.md) will be generated.
+	 */
+	"apiReport": {
+		"enabled": true,
+		"reportFileName": "<unscopedPackageName>.api.md",
+		"reportFolder": "<projectFolder>/api-report/",
+		"reportTempFolder": "<projectFolder>/_api-extractor-temp/"
+	},
+
+	/**
+	 * Configures how the doc model file (*.api.json) will be generated.
+	 */
+	"docModel": {
+		"enabled": true,
+		"apiJsonFilePath": "<projectFolder>/_api-extractor-temp/doc-models/<unscopedPackageName>.api.json"
+	},
+
+	/**
+	 * Configures how the .d.ts rollup file will be generated.
+	 */
+	"dtsRollup": {
+		"enabled": true,
```

Comment:
I think we should set the default `dtsRollup - enabled` property to false. This updates alpha/beta types in all the packages. example run: https://dev.azure.com/fluidframework/internal/_build/results?buildId=200523&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=3e8a0811-333e-5612-55d7-750fabab9520

any reason why this is set to true as default?

Reply:
There isn't a reason to disable this behavior IMO. Packages can opt out if they want to. But I expect most of our packages will use these, and in the meantime it's harmless to generate them.